src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/InitializeHandler.cs

```diff
@@ -26,6 +26,11 @@ internal class InitializeHandler : IRequestHandler<InitializeParams, InitializeR
                 {
                     TriggerCharacters = new[] { ">", "=", "-" }
                 },
+                DocumentOnTypeFormattingProvider = new DocumentOnTypeFormattingOptions()
```

Comment:
Should leave a comment here stating why we can't overlap with OnAutoInsert pieces due to the VSTS issue.